Library/Homebrew/utils/string_inreplace_extension.rb

```diff
@@ -27,7 +31,7 @@ def sub!(before, after)
   #
   # @api public
   sig {
-    params(before: T.any(Pathname, Regexp, String), after: T.nilable(String), audit_result: T::Boolean)
+    params(before: T.any(Regexp, String), after: String, audit_result: T::Boolean)
       .returns(T.nilable(String))
   }
   def gsub!(before, after, audit_result = true) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
The Loki formula invokes `gsub!` directly with pathnames

Reply:
Sorry for the regression, PTAL. (I've also attempted to add a spec based on the Loki formula, to prevent this regression in the future)